### ISSUE TITLE
Wordwrap onto a new line instead of breaking a command

### ISSUE THREAD

**Issue by rs7q5** [James-Yu/LaTeX-Workshop#3542]
There may be a setting but I could not find it. I was wondering if it is possible to have a setting where when using wordwrap, it trys not to break a command across lines if it does not have to.

For example if `\cite{reference}` as at the end of the window or near the end of the line then in TexStudio it will move the entire command to the next line if it will fit, but in VSCode it will break it at `\cite` and then the next line starts with `{\reference}`. I would like it so `\cite{reference}` just gets put on the new line. 

Current wordwrap setttings for latex files in VSCode:

![image](https://user-images.githubusercontent.com/46969733/202050580-345a7fe8-358f-40ac-9716-8b40c7714f28.png)

**Comment by James-Yu** [comment]
Correct me if I am wrong but I don't think the word-break can be defined by the extension.

**Comment by jlelong** [comment]
The `wordPattern` entry is a very sensitive setting with a lot of side effects. It impacts line wrapping, snippet suggestions, intellisense triggering and filtering, and bracket pair matching at least. Every change turned out to be very tricky. I am really not in favour of fiddling with this setting.

**Comment by rs7q5** [comment]
I thought I may be able to use a setting like `word Separators`, but changes to that did not seem to work. 

Am I bound by word pattern or word separators, I changed word separators and it did not seem to help, but when I searched `wordPattern` when changing language specific settings the search did not come up with anything. 

Link below says `wordPattern` does nothing if the language is controlled by `wordSeparators`
<https://code.visualstudio.com/api/language-extensions/language-configuration-guide>

Is there a way to capture groups or manually add certain commands? I don't think you'd ever get all of the commands right off the bat anyways. 

I do understand the hesitation to change the setting, but am willing to try something locally if necessary too.
